querybook/server/lib/query_executor/executors/trino.py

```diff
@@ -36,22 +36,24 @@ def _parse_exception(self, e):
         error_type = QueryExecutionErrorType.INTERNAL.value
         error_str = str(e)
         error_extracted = None
-        try:
-            if isinstance(e, Error):
-                error_type = QueryExecutionErrorType.ENGINE.value
+
+        if isinstance(e, TrinoQueryError):
+            try:
+                line_number, column_number = e.error_location
```

Comment:
so this is to guard e.error_location to ensure it returns a tuple right? just making sure